ndoh_hub/utils.py

```diff
@@ -118,18 +118,16 @@ def deactivate_subscription(subscription):
     return patch_subscription(subscription, {"active": False})
 
 
-def get_messageset_short_name(recipient, stage, authority):
-    # Examples:
-    # prebirth.mother.hw_full
-    # prebirth.household.patient
+def get_messageset_short_name(reg_type, authority, weeks):
+    set_number = 1  # default set_number
```

Comment:
can we call the `batch` ... because set has another meaning in this context and using integers it might get complicated.